lib/honeybadger/breadcrumbs/active_support.rb

```diff
@@ -104,6 +104,16 @@ def self.default_notifications
           }
         }
       end
+
+      private_class_method def self.active_record_connection_db_config
+        if ::ActiveRecord::Base.respond_to?(:connection_db_config)
+          # >= Rails 6.1
```

Comment:
Are these comments accurate? I probably added them, but based on your PR description, I'm doubting myself. :)

Reply:
Looks like `connection_db_config` went in 6.0.x, but I think `connection_config` will be removed in 6.1.